src/fmu/dataio/aggregation.py

```diff
@@ -230,32 +232,28 @@ def _generate_aggrd_metadata(
         template["fmu"]["context"]["stage"] = "iteration"
 
         # next, the new object will trigger update of: 'file', 'data' (some fields) and
-        # 'tracklog'. The trick is to create an ExportData() instance and just retrieve
-        # the metadata from that, and then blend the needed metadata from here into the
-        # template -> final metadata
-
-        fakeconfig = {
-            "access": self.source_metadata[0]["access"],
-            "masterdata": self.source_metadata[0]["masterdata"],
-            "model": self.source_metadata[0]["fmu"]["model"],
-        }
-
+        # 'tracklog'.
         content = template["data"]["content"]
-        etemp = dataio.ExportData(config=fakeconfig, name=self.name, content=content)
-        etempmeta = etemp.generate_metadata(obj, compute_md5=compute_md5)
+        etemp = dataio.ExportData(name=self.name, content=content)
```

Comment:
Removed the fakeconfig as argument here now since we don't need to generate metadata in order to populate the necessary fields.